packages/aws-cdk-lib/aws-kms/lib/alias.ts

```diff
@@ -67,6 +73,15 @@ abstract class AliasBase extends Resource implements IAlias {
     });
   }
 
+  /**
+   * The ARN of the alias.
+   *
+   * @attribute
+   */
+  public get aliasArn(): string {
+    return this.keyArn;
+  }
```

Comment:
Can we just copy the logic here? No need to make an additional function call to a now-deprecated function.

Reply:
@scanlonp was thinking about this, but would be good to have duplicated code? Or should we move the logic to the `aliasArn` and then call it in `keyArn`?

To be honest both should work and be consistent because of the test (`aliasArn and keyArn from alias should match`), but it's something to think about.

I can get to this as soon as I receive feedback on this.

Reply:
I think having the same code in both places is best here. Minimally better for performance, and also clearer to understand. 
Also if we ever remove the deprecated function in a major version, there does not need to be any mental effort other than simply removing it.